### PR TITLE
Upgraded app to Ember v3.20

### LINKED ISSUE #39: Spike: Add line numbers to code snippets
## Background

In #38, @Wolfr and I discussed how adding horizontal scrolling to unwrap a long line of code can result in a negative user experience. (Not easy to scroll left and right and remember the entire code.)

We think it may be good to try adding line numbers instead, so that when a line of code does wrap around, a user can more easily distinguish that line from the next.

In my initial research, I saw that `ember-code-snippet` and `ember-prism` don't provide a component argument to enable line numbers. I think we can add line numbers by using a Prism plugin instead: https://prismjs.com/plugins/line-numbers/

## Suggested action items

Let's do a spike and investigate if it's easy to install the `line-numbers` plugin without having to make many code changes. If it is, we can make a PR for review and merge. If not, I think we can close this issue.

### PR DESCRIPTION
**Pull request by ijlee2**
## Description

This PR makes 2 improvements:

- Upgrades `ember-source` from v3.19 to v3.20
- Upgrades `ember-prism` to v0.8 to fix a splattributes bug and show line numbers

In particular, the second item addresses and can close #39. (Many thanks to @rwwagner90 for helping with addressing the splattributes bug and making a new release.)

## Screenshots

Local server:

screenshot

Percy snapshot:

percy_snapshot

There seems to be a height issue in our app such that Percy doesn't correctly render the snapshot. The issue may also be related to `ember-qunit` resizing the window. I don't think we need to worry about the snapshot much at the moment.

### CODE REVIEW

File: config/targets.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
-
 module.exports = {
-  browsers
+  browsers: [
```
Comment (ijlee2) [inline config/targets.js]: At work, I found that allowing `last 1 versions` can lead to unexpected build errors. I set these targets based on:

- http://kangax.github.io/compat-table/es2016plus/
- https://caniuse.com/usage-table

For this app, I think it's okay to drop support for IE 11.

File: config/targets.js [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
+    'Chrome >= 83',
+    'Edge >= 18',
+    'Firefox >= 77',
+    'Safari >= 13'
+  ],
+  node: 'current'
```
Comment (ijlee2) [inline config/targets.js:18]: Based on https://github.com/ember-fastboot/ember-cli-fastboot/pull/770

File: config/targets.js (hunk 1 above)
Comment (RobbieTheWagner) [reply config/targets.js]: I would recommend not moving away from the default Ember blueprints on these targets.

**Review by RobbieTheWagner — CHANGES_REQUESTED** [review]
Just one thing about targets, but the rest looks good!

File: config/targets.js (hunk 1 above)
Comment (ijlee2) [reply config/targets.js]: Ah, ok. I'll revert them.

Edit: I removed the commit that changed the browser targets.

**Review by RobbieTheWagner — APPROVED** [review]

**Pull request merged**